### PR TITLE
[RV64_DYNAREC] Added more opcodes ([LA64_DYNAREC] too)

### PR DESCRIPTION
**Pull request by ksco**

### CODE REVIEW

File: src/dynarec/rv64/dynarec_rv64_66.c [hunk 1]
```diff
@@ -718,6 +718,21 @@ uintptr_t dynarec64_66(dynarec_rv64_t* dyn, uintptr_t addr, uintptr_t ip, int ni
                 SMWRITE2();
             }
             break;
+        case 0x8E:
+            INST_NAME("MOV Seg, Ew");
+            nextop = F8;
+            u8 = (nextop & 0x38) >> 3;
+            if ((nextop & 0xC0) == 0xC0) {
```
Comment (ptitSeb) [inline src/dynarec/rv64/dynarec_rv64_66.c:725]: isn't there a MODREG  macro for that?

File: src/dynarec/la64/dynarec_la64_66.c [hunk 2]
```diff
@@ -497,6 +497,21 @@ uintptr_t dynarec64_66(dynarec_la64_t* dyn, uintptr_t addr, uintptr_t ip, int ni
                 BSTRINSz(gd, x1, 15, 0);
             }
             break;
+        case 0x8E:
+            INST_NAME("MOV Seg, Ew");
+            nextop = F8;
+            u8 = (nextop & 0x38) >> 3;
+            if ((nextop & 0xC0) == 0xC0) {
```
Comment (ptitSeb) [inline src/dynarec/la64/dynarec_la64_66.c:504]: `if(MODREG)` ?

File: src/dynarec/la64/dynarec_la64_00.c [hunk 3]
```diff
@@ -1137,6 +1137,18 @@ uintptr_t dynarec64_00(dynarec_la64_t* dyn, uintptr_t addr, uintptr_t ip, int ni
                 LDxw(gd, ed, fixedaddress);
             }
             break;
+        case 0x8C:
+            INST_NAME("MOV Ed, Seg");
+            nextop = F8;
+            if ((nextop & 0xC0) == 0xC0) { // reg <= seg
```
Comment (ptitSeb) [inline src/dynarec/la64/dynarec_la64_00.c:1143]: MODREG

File: src/dynarec/rv64/dynarec_rv64_66.c (hunk 1 above)
Comment (ksco) [reply src/dynarec/rv64/dynarec_rv64_66.c:725]: Please merge this PR, I will do a global replacement in another PR, there are many places need to be replaced across all 3 backends.

Comment (ptitSeb) [reply src/dynarec/rv64/dynarec_rv64_66.c:725]: ok

**Pull request merged**